src/components/molecules/UiLoader/_internal/UiLoaderTransition.vue

```diff
@@ -21,49 +21,49 @@ import {
   onMounted,
 } from 'vue';
 import type { CSSProperties } from 'vue';
+import type { DefineAttrsProps } from '../../../../types';
 
-export type TransitionState = 'from' | 'active' | 'to';
-export type Move = 'leave' | 'enter';
-export type TransitionHook = `before-${Move}`
-  | `${Move}`
-  | `after-${Move}`
-  | `${Move}-cancelled`;
-const props = defineProps({
+export type LoaderTransitionState = 'from' | 'active' | 'to';
+export type LoaderTransitionMove = 'leave' | 'enter';
+export type LoaderTransitionHook = `before-${LoaderTransitionMove}`
```

Comment:
Nice! It could be simplified even more, like
```typescript
export type LoaderTransitionHook = `${'before' | 'after'}-${LoaderTransitionMove}`
  | `${LoaderTransitionMove}`
  | `${LoaderTransitionMove}-cancelled`;
```